package.json

```diff
@@ -37,8 +37,7 @@
     "ibm_db": "2.7.1"
   },
   "peerDependencies": {
-    "@zowe/imperative": "^4.0.0",
-    "@zowe/cli": "^6.0.0"
+    "@zowe/imperative": "^4.0.0"
```

Comment:
Should this versions be have a prerelease string (since this is going inot Next)?